app/controllers/manager/venues_controller.rb

```diff
@@ -2,6 +2,7 @@ class Manager::VenuesController < ApplicationController
 
   before_action :set_venue, only: [:edit, :show, :update]
   before_action :require_user
+  before_action :require_correct_manager, only: [:edit, :show, :update]
```

Comment:
Is there ever a circumstance where we wouldn't want to `require_correct_manager` in the manager/venues controller?